trunk/src/GMP/base/input_DEFAULT.F90

```diff
@@ -221,12 +227,30 @@ subroutine input_DEFAULT(Fp)
       read(nin,*) NRDOMAIN
 !
 !---------------------------------------------------------------------
-!  POINTS                                                            |
+!  POINTS  and surface domains flag                                  |
 !---------------------------------------------------------------------
 !
 !  ...read in number of points
       read(nin,*) NRPOINT
- IF (.NOT. QUIET_MODE)     write(*,1009) NRPOINT
+!
+!  ...most likely, we have read the surface domains flag
+      if ((NRPOINT.eq.0).or.(NRPOINT.eq.1)) then
+        isurf_flag = NRPOINT
```

Comment:
uses 2 space indentation instead of standard 3.

Reply:
Yeah, it was just kept consistent with the rest of the file which currently has two-space indentation.